README.md

```diff
@@ -4,7 +4,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 -update
 
 In the project directory, you can run:
-
+-- update here for semaphore test
```

Comment:
?

Reply:
When we were working with Faraz, he wanted me to try pushing up again to trigged semaphore to rebuild so we added some text to the readme file as a test. I can remove that if you'd like